src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesImplementation.cs

```diff
@@ -1,15 +1,48 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
 using Internal.Cryptography;
 
 namespace System.Security.Cryptography
 {
     internal sealed partial class AesImplementation : Aes
     {
+        private FixedMemoryKeyBox? _keyBox;
```

Comment:
I don't see this get disposed anywhere except when we replace the key with a new one. Are we relying on finalizers to clean these up?

Reply:
Fixed.

Long-winded version: When I initially started the change I hadn't written UseKey, and was just blithely using the DangerousGimmeYourSpan accessor, and realized that parallel dispose was going to toast things, so ended up deciding "can't dispose".  Then decided that was both stupid and contrary to the goals, so wrote UseKey... and just missed this, the most obvious place :smile: